### ISSUE TITLE
Improve description of project

### ISSUE THREAD

**Issue by peterfarrell** [divio/djangocms-inherit#10]
I honestly didn't really understand what the practical application of this app was until I read through some source code.  It would be great if the README was updated to better reflect what `django-inherit` actually is.

**Comment by bittner** [comment]
:+1:

Also, the following paragraph in the README is outdated and should be replaced by appropriate instructions for Django 1.7 and 1.8: (bullet no. 3)
```
...; when django CMS 3.1 will be released, migrations for Django 1.7 will be moved to ...
```
After all, django CMS has been released for a while, hasn't it?

This is *not* a *non*-critical request: It's a pain getting this plugin running with Django 1.8, requiring significant research effort for a solution.

**Comment by mkoistinen** [comment]
Resolved by #19.
